demo/toast.py

```diff
@@ -0,0 +1,182 @@
+"""Simple toast component that is similar to Angular Component Snackbar."""
```

Comment:
I'd probably just call this snackbar to be consistent with what material calls it (i.e. https://m3.material.io/components/snackbar/overview), I think toast is an old android UI/API which looks a little different (there's no action/button) than snackbar, which is more modern - https://www.geeksforgeeks.org/differnce-between-android-toast-and-snackbar/

Reply:
Lol, thanks. I didn't realize they were different UI things. Updated.